it/uk/gov/hmrc/agentmappingfrontend/controllers/MappingControllerISpec.scala

```diff
@@ -4,19 +4,18 @@ import play.api.http.Writeable
 import play.api.mvc.{Request, Result}
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
-import uk.gov.hmrc.agentmappingfrontend.model.Identifier
+import uk.gov.hmrc.agentmappingfrontend.auth.Auth
 import uk.gov.hmrc.agentmappingfrontend.stubs.AuthStubs
 import uk.gov.hmrc.agentmappingfrontend.stubs.MappingStubs.{mappingExists, mappingIsCreated, mappingKnownFactsIssue}
-import uk.gov.hmrc.agentmappingfrontend.support.SampleUsers.{anSAEnrolledAgent, anVATEnrolledAgent, anAgentNotEnrolled}
+import uk.gov.hmrc.agentmappingfrontend.support.SampleUsers.{aValidlyEnrolledAgent, anVATEnrolledAgent}
 import uk.gov.hmrc.agentmtdidentifiers.model.{Arn, Utr}
-import uk.gov.hmrc.domain.SaAgentReference
 
 class MappingControllerISpec extends BaseControllerISpec with AuthStubs {
 
   def callEndpointWith[A: Writeable](request:Request[A]): Result = await(play.api.test.Helpers.route(app, request).get)
 
   "context root" should {
-    "redirect to start page" in {
+    "redirect to the start page" in {
       val request = FakeRequest(GET,"/agent-mapping/")
       val result = callEndpointWith(request)
       status(result) shouldBe 303
```

Comment:
redirectLocation shouldBe
will check where you been redirected too, and that its 303